src/generate-changelog.ts

```diff
@@ -10,7 +10,8 @@ const TEAMS_SPECIFIC_REGEX = /^TEAMS-SPECIFIC-CHANGES:(.*)/gm;
 // Template text for the changelog that should be ignored.
 const CHANGELOG_TEMPLATE_TEXT = [
   '{{Insert a single changelog entry here}}',
-  '{{Insert ANOTHER changelog entry here}}'
+  '{{Insert ANOTHER changelog entry here}}',
+  '{{Insert a single team-related changelog entry here}}',
```

Comment:
This fixes part of it, but it will _still_ show teams features to preview users regardless of whether they have access to the features. Just confirming, you're also tracking fixing that part as well?

Reply:
Yep!

Reply:
awesome, ty!